### ISSUE TITLE
The Tailon scrollback buffer should be larger and/or configurable

### ISSUE THREAD

**Issue by rcourtna** [gvalkov/tailon-legacy#2]
This is an enhancement request.

I find with the logs that I tail, the scrollback buffer is too small. Would be great if it was larger, or even better, configurable.

**Comment by gvalkov** [comment]
Thanks for the suggestion. I'll definitely look into it when I get the time to hack on tailon. There are certainly a lot of things that need to be made configurable.

**Comment by malford** [comment]
+1

It would be great if it were possible to change the `tail-lines` default from 60. Any chance of getting an enhancement? :grinning:

**Comment by gvalkov** [comment]
Thank you for the reminder @malford. I've been meaning to implement the configuration part of the UI, but between work and home I just never get around to doing it. It's on my backlog for sure :(

**Comment by schrej** [comment]
Would be really awesome. I'm also trying it out atm and its really helpful. But the scrollback buffer is really small. Configuration would be really great!

**Comment by gvalkov** [comment]
Gentlemen, I hacked together a configuration dialog that would allow you to change the scrollback buffer size as well as the number of lines to tail. It looks like this:

![2015-09-08-005511_365x186_scrot](https://cloud.githubusercontent.com/assets/190997/9723600/c78d73d6-55c4-11e5-8a18-9c8ee36676b7.png)

I hope to release a new version by the of the week.

Cheers,
G.

**Comment by vinaybalepur** [comment]
This is awesome. You guys rock. Let me know when the latest code is
available for download.

On Tue, Sep 8, 2015 at 4:31 AM, Georgi Valkov notifications@github.com
wrote:

> Gentlemen, I hacked together a configuration dialog that would allow you
> to change the scrollback buffer size as well as the number of lines to
> tail. It looks like this:
> 
> [image: 2015-09-08-005511_365x186_scrot]
> https://cloud.githubusercontent.com/assets/190997/9723600/c78d73d6-55c4-11e5-8a18-9c8ee36676b7.png
> 
> I hope to release a new version by the of the week.
> 
> Cheers,
> G.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/gvalkov/tailon/issues/2#issuecomment-138389929.

## 

_Thanks and regards,       Vinay_

**Comment by gvalkov** [comment]
Fixed in version 0.4.0. Please reopen if anything doesn't work as expected.

**Comment by vinaybalepur** [comment]
Hi,

```
    The feature seems to be working as expected. Awesome help. Thank
```

you.

On Wed, Sep 9, 2015 at 3:20 AM, Georgi Valkov notifications@github.com
wrote:

> Closed #2 https://github.com/gvalkov/tailon/issues/2.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/gvalkov/tailon/issues/2#event-404184763.

## 

_Thanks and regards,       Vinay_
